moveit_ros/visualization/planning_scene_rviz_plugin/include/moveit/planning_scene_rviz_plugin/planning_scene_display.h

```diff
@@ -112,6 +112,9 @@ class PlanningSceneDisplay : public rviz::Display
   planning_scene_monitor::LockedPlanningSceneRW getPlanningSceneRW();
   const planning_scene_monitor::PlanningSceneMonitorPtr& getPlanningSceneMonitor();
 
+Q_SIGNALS:
+  void changeSceneName(const QString& name);
```

Comment:
```suggestion
  void sceneNameChanged(const QString& name);
```
would be following the Qt naming scheme

Reply:
MoveIt uses the naming scheme `changedXXX`:
https://github.com/ros-planning/moveit/blob/38852558d749a029826e827cf9165a5b458b6e87/moveit_ros/visualization/planning_scene_rviz_plugin/include/moveit/planning_scene_rviz_plugin/planning_scene_display.h#L123-L125

However, in contrast to the _passive notification_ slot `changedSceneName()`, the new slot `changeSceneName(QString)` will `actively` change the name.

Reply:
Ok good. But still this is a `signal` where I suggested the change so it should be _passive_ ?

Reply:
Also I cannot find a `connect` in that diff that actually uses the `signal` `changeSceneName`?

Reply:
> But still this (`changeSceneName`) is a `signal` where I suggested the change so it should be _passive_ ?

Maybe, it should be named `triggerSceneNameChange` ...

> Also I cannot find a `connect` in that diff that actually uses the `signal` `changeSceneName`?

Good catch! I missed adding this in a hurry. Fixed now.

Reply:
You can also call slots directly with `invokeMethod` if your signal is not actually a signal ...